frontend/src/pages/home.js

```diff
@@ -14,7 +15,14 @@ function Home() {
         setBackendData(data)
       }
     )
-  }, []);
+    //check if user is logged in
+    const loggedInUser = localStorage.getItem('user');
+    if (loggedInUser) {
+      setIsLoggedIn(true);
+    } else {
+      setIsLoggedIn(false);
+    }
+  }, [setIsLoggedIn, setBackendData]);
```

Comment:
check this one also (it should be the same changes as the previous one I think)

Reply:
fixed